cmd/codegen/gencpp.go

```diff
@@ -266,32 +275,32 @@ extern "C" {
 
 		invokeFunctionName := f.FuncName
 		if len(f.OriginalFuncName) > 0 {
-			invokeFunctionName = f.OriginalFuncName
+			invokeFunctionName = "wrap_" + f.OriginalFuncName
```

Comment:
just wondering: why do you add this `wrap_` prefix? did it collide with something? If so, maybe write a comment somewhere

Reply:
Yes, some of them collide. I will add a comment.